frontend/src/metabase-types/api/bookmark.ts

```diff
@@ -1,9 +1,11 @@
-export type BookmarkableEntities = "card" | "collection";
+export type BookmarkableEntities = "card" | "collection" | "dashboard";
 
 export interface Bookmark {
+  authority_level?: string;
```

Comment:
`authority_level` can be `null` too

Reply:
@kulyk This is what I'm seeing

<img width="500" alt="image" src="https://user-images.githubusercontent.com/380816/160153724-bb5440ee-802d-4579-901f-aac9e07f08f1.png">

In other words, `authority_level` is either set to "official" or undefined.

Do you know any cases when `authority_level` is passed as null, in the bookmarks list?